devices/SparkfunLCD/samples/Program.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.SparkfunLCD.sample
+{
+    using System.Device.I2c;
+    using System.Diagnostics;
+    using System.Threading;
+    using nanoFramework.Hardware.Esp32;
+    using Iot.Device.SparkfunLCD;
+
+    /// <summary>
+    /// Class containing main executable code
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// code executed at device reset
+        /// </summary>
+        public static void Main()
+        {
+            Debug.WriteLine("Hello from SparkFun 20x4 SerLCD");
+
+            //// using (var lcd = new SparkfunLCD(SparkfunLCD.DISPLAYSIZE.SIZE20X4, Gpio.IO23, Gpio.IO22))
```

Comment:
Based on the requested changes, you'll have to setup the I2cDevice here and set the pins here.